### ISSUE TITLE
is there any way or chance to use flash_attn 1.x ? to support more gpus

### ISSUE THREAD

**Issue by dalamudx** [vectorch-ai/ScaleLLM#69]
i think it depend on flash_attn 2.x,will ouput runtime error with unsupport gpus.

**Comment by guocuimi** [comment]
Yes, you are correct. Currently, we only support Ampere or newer architecture. The flash_attn1 implementation does indeed work for Turing GPUs. At the moment, we don't have plans to integrate flash_attn1, but we do intend to extend support to more GPUs. One of our ongoing efforts is to integrate [flashinfer](https://github.com/flashinfer-ai/flashinfer/) as an attention implementation, which supports Turing gpu and shows promising perf results. thanks, stay tuned.

**Comment by dalamudx** [comment]
> Yes, you are correct. Currently, we only support Ampere or newer architecture. The flash_attn1 implementation does indeed work for Turing GPUs. At the moment, we don't have plans to integrate flash_attn1, but we do intend to extend support to more GPUs. One of our ongoing efforts is to integrate [flashinfer](https://github.com/flashinfer-ai/flashinfer/) as an attention implementation, which supports Turing gpu and shows promising perf results. thanks, stay tuned.

that would be great!thanks for your work😀
